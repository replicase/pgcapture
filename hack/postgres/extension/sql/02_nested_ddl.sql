LOAD 'pgcapture';
TRUNCATE pgcapture.ddl_logs;

DO $$
BEGIN
    CREATE TABLE tbl1();
    CREATE TABLE tbl2();
END;
$$ LANGUAGE plpgsql;

CREATE EXTENSION pg_stat_statements;

-- Check the results
SELECT query, unnest(tags) FROM pgcapture.ddl_logs ORDER BY id;
