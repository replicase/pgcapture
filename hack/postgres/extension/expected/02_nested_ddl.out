LOAD 'pgcapture';
TRUNCATE pgcapture.ddl_logs;
DO $$
BEGIN
    CREATE TABLE tbl1();
    CREATE TABLE tbl2();
END;
$$ LANGUAGE plpgsql;
CREATE EXTENSION pg_stat_statements;
-- Check the results
SELECT query, unnest(tags) FROM pgcapture.ddl_logs ORDER BY id;
                query                |      unnest      
-------------------------------------+------------------
 CREATE TABLE tbl1()                 | CREATE TABLE
 CREATE TABLE tbl2()                 | CREATE TABLE
 CREATE EXTENSION pg_stat_statements | CREATE EXTENSION
(3 rows)

